src/data/mv_polynomial/basic.lean

```diff
@@ -1099,6 +1099,9 @@ section aeval
 variables [algebra R S₁] [comm_semiring S₂]
 variables (f : σ → S₁)
 
+lemma algebra_map_apply (r : R) :
```

Comment:
Please don't forget to forward-port this change :)

Reply:
https://github.com/leanprover-community/mathlib4/pull/2954